src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1251,18 +1259,28 @@ public function process_posts( $data = [], $start_immediately = false ) {
 			$service->confirm_import( $this->meta );
 		}
 
+		// CSV should be processed right away as does not have support for batch pushing.
+		$is_not_csv = empty( $data ) || empty( $data['origin'] ) || $data['origin'] !== 'csv';
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 4e31ed941bb9419b761453fb5690d98f</sub>